dot/state/epoch.go

```diff
@@ -47,6 +51,18 @@ func configDataKey(epoch uint64) []byte {
 	return append(configDataPrefix, buf...)
 }
 
+func nextEpochDataKey(epoch uint64, hash common.Hash) []byte {
+	// we add a "-" to the key to avoid conflicts with composite keys
+	key := fmt.Sprintf("-%d:%v", epoch, hash)
```

Comment:
would suggest to use `partialKey` since the full key contains the `nextEpochDataPrefix`
```suggestion
	partialKey := fmt.Sprintf("-%d:%v", epoch, hash)
```